src/libraries/System.Data.Odbc/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,363 @@
+    <Left>lib/net6.0/System.Data.Odbc.dll</Left>
+    <Right>runtimes/win/lib/net6.0/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.set_Offset(System.Int32)</Target>
+    <Left>lib/net6.0/System.Data.Odbc.dll</Left>
+    <Right>runtimes/win/lib/net6.0/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/freebsd/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.set_Offset(System.Int32)</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/freebsd/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/linux/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.set_Offset(System.Int32)</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/linux/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/osx/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.set_Offset(System.Int32)</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/osx/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/win/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.set_Offset(System.Int32)</Target>
+    <Left>lib/netcoreapp3.1/System.Data.Odbc.dll</Left>
+    <Right>runtimes/win/lib/netcoreapp3.1/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Data.Odbc.OdbcCommand</Target>
+    <Left>lib/netstandard2.0/System.Data.Odbc.dll</Left>
+    <Right>lib/net461/System.Data.Odbc.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
```

Comment:
We should be including a comment that refers to an issue for these type-forward related suppressions so that we remember to clean them up.

Reply:
I created an issue to track cleaning them up https://github.com/dotnet/runtime/issues/56823. I'm hesitant to adding comments in these files as @Anipik there was a bug around that and I don't know when it was exactly fixed.

Reply:
Ok, I think the issue should suffice.